mat64/cblas_test.go

```diff
@@ -0,0 +1,15 @@
+// Copyright ©2015 The gonum Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+//+build cblas
+
+package mat64
+
```

Comment:
Use the cblas panic messages rather than the goblas ones? SGTM.

I see what this code does now. Thanks. I hadn't used build tags before.